docs/software/plugins/environment.md

```diff
@@ -5,7 +5,7 @@ sidebar_label: Environment measurement
 ---
 ## About
 
-The Environment Measurement Plugin will allow nodes to send a specific message with information from connected environmental sensors. Currently supported sensors are BME280, BME680, DHT11, DHT12, DHT21, DHT22 and Dallas 1-wire DS18B20.
+The Environment Measurement Plugin will allow nodes to send a specific message with information from connected environmental sensors. Currently supported sensors are BME280, BME680, MCP9808, DHT11, DHT12, DHT21, DHT22 and Dallas 1-wire DS18B20.
```

Comment:
Currently this list is alphabetized. Could you switch this:
```
Currently supported sensors are BME280, BME680, MCP9808, DHT11, DHT12, DHT21, DHT22 and Dallas 1-wire DS18B20.
```
to this:
```
Currently supported sensors are BME280, BME680, DHT11, DHT12, DHT21, DHT22, Dallas 1-wire DS18B20, and MCP9808.
```

Thanks!